integration-tests/testconfig/ccip/config.go

```diff
@@ -45,6 +49,38 @@ type RMNConfig struct {
 	AFNVersion   *string `toml:",omitempty"`
 }
 
+func (r *RMNConfig) GetProxyImage() string {
+	image := pointer.GetString(r.ProxyImage)
+	if image == "" {
+		return ctfconfig.MustReadEnvVar_String(E2E_RMN_RAGEPROXY_IMAGE)
+	}
+	return image
+}
+
+func (r *RMNConfig) GetProxyVersion() string {
+	version := pointer.GetString(r.ProxyVersion)
+	if version == "" {
+		return ctfconfig.MustReadEnvVar_String(E2E_RMN_RAGEPROXY_VERSION)
+	}
+	return version
+}
+
+func (r *RMNConfig) GetAFNImage() string {
+	image := pointer.GetString(r.AFNImage)
+	if image == "" {
+		return ctfconfig.MustReadEnvVar_String(E2E_RMN_AFN2PROXY_IMAGE)
+	}
+	return image
+}
+
+func (r *RMNConfig) GetAFNVersion() string {
```

Comment:
Shouldn't we align with a new naming? I guess no one remembers what AFN was :P

Reply:
The image name is AFN2Proxy https://github.com/smartcontractkit/rmn-v2-test-infra/tree/master/docker/afn2proxy